src/components/Input/Input.tsx

```diff
@@ -32,7 +32,6 @@ const Input: FC<InputElementProps> = ({
 					<option
 						className={classes.inputItem}
 						disabled
-						selected
 						value={value}
```

Comment:
no idea how this change is relevant to this PR

Reply:
Few daus ago you have asked me to check errors in console and find some solution.
I anderstand: this change possibly should be in other PR, but shis error is really annoing to me/
Thats why io`ve dicided to leave it here
![image](https://github.com/user-attachments/assets/0f1b4d22-b37d-4434-9f5a-d6a62e91599c)